config/rbac/extension_viewer_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to view extensions.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: extension-viewer-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: operator-controller
+    app.kubernetes.io/part-of: operator-controller
+    app.kubernetes.io/managed-by: kustomize
```

Comment:
Nit: drop all these labels (here and in the editor role). These are setup by SDK/Kubebuilder, right? I don't think they provide any real value.

Reply:
If I remove them, they might be re-added?

Reply:
I'm guessing this is part of `operator-sdk create api`? So no they would not be re-added.

Reply:
Not sure, but I'll try; there definitely a build process that regenerates _some_ of these types of things.